### ISSUE TITLE
Confusion regarding rotated annotations

### ISSUE THREAD

**Issue by adityasihag1804** [NVIDIA/retinanet-examples#333]
I'm trying to do rotated detections on text pages. It is a custom dataset. I'm confused regarding the `bbox` values and `segmentation` values. here's a sample image from the training dataset, with a few `axis-aligned` (Blue), also the minimum spanning rectangle, and `rotated` boxes (green).

<img width="544" alt="Screenshot 2022-05-11 at 1 14 10 PM" src="https://user-images.githubusercontent.com/87427616/167796671-620b7738-f484-4e57-92d8-a11e99d15aed.png">

So for the field `bbox` , `x y w h` , I have tried passing the values for axis-aligned and rotated both.
And for `segmentation` , I'm always passing the 4 coordinates of the green box.
But the model seems to not be learning anything. The loss seems to be decreasing, but mAP remains 0 the entire time. Please let me know if I'm doing something wrong here.
